assets/src/components/SurveyModal.js

```diff
@@ -41,6 +41,10 @@ const useStyles = makeStyles((theme) => ({
     right: 0,
     width: '100%',
     height: '100%'
+  },
+  surveyButton: {
```

Comment:
I played around with adding `fontWeight: bold`. More noticeable, but also kinda irritating looking. I guess I'd lean towards what you have, but you could try it out and see what you think.

Reply:
Yeah, I don't really like it that much and it doesn't seem to add much. I think once we get the custom text in here it should look good.

Reply:
Fine with me.